src/PlatesRendererFactory.php

```diff
@@ -39,11 +43,17 @@ class PlatesRendererFactory
     public function __invoke(ContainerInterface $container)
     {
         $config = $container->has('config') ? $container->get('config') : [];
-        $config = isset($config['templates']) ? $config['templates'] : [];
 
         // Create the engine instance:
         $engine = new PlatesEngine();
 
+        // Enable assets extension
+        if (isset($config['plates']['assets_path'])) {
+            $engine->loadExtension(new PlatesExtension($config['plates']['assets_path']));
```

Comment:
I understand, I'll work that out. I was trying to study the twig code, still need to get my head around it. Will update soon.